Client/mods/deathmatch/logic/CClientGame.cpp

```diff
@@ -2325,6 +2329,58 @@ void CClientGame::ProcessServerControlBind(CControlFunctionBind* pBind)
     m_pNetAPI->RPC(KEY_BIND, bitStream.pBitStream);
 }
 
+CClientMarker* CClientGame::CheckMarkerClick(float screenX, float screenY, float& distance) noexcept
```

Comment:
You should only use noexcept if the functions being called are also noexcept, so in this case these functions should not be marked as noexcept.